decidim/decidim
# Import proposal answers

#### :tophat: What? Why?
Imports proposal answers from file (excel, csv, json).

#### :pushpin: Related Issues
https://meta.decidim.org/processes/roadmap/f/122/proposals/14922

#### Testing
1. Create .xlsx file, colums: id, state, answer/en (depends on organization locales)
2. Write rows to excel. Example row: 1, accepted, This is proposal is very nice. 
3. Go to admin panel -> components -> Proposals  (component must match proposal's component!) -> Import (dropdown) -> Import proposal answers from a file
4. Upload file and select "Import"
5. Check proposal answers

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/131837657-3ff1cbed-e794-4bf9-a0ef-82fd15c3ef02.png)

![image](https://user-images.githubusercontent.com/19709320/131837675-4d1ebe38-8c1f-46ab-8710-08f3bb8a92c9.png)

:hearts: Thank you!
